source/configuration_manual/namespace/index.rst

```diff
@@ -555,109 +563,156 @@ on and setting ``subscriptions = no`` for the other namespaces:
+other recipients.
 
-  .. code-block:: none
+doveadm and other processes use this as well. If you want to limit expunging
+to only user sessions, the setting can be defined only within certain
+protocols (e.g. imap, pop3), such as::
 
-    protocol imap {
-      namespace inbox {
-	mailbox Spam {
-	  autoexpunge = 10d
-	}
+  protocol imap {
+    namespace inbox {
+      mailbox Spam {
+        autoexpunge = 10d
       }
     }
+  }
 
+":ref:`setting-mailbox_list_index` = yes" is highly recommended when using
+this setting, as it avoids actually opening the mailbox to see if anything
+needs to be expunged.
 
-* mailbox_list_index=yes is highly recommended when using this setting, as it
-  avoids actually opening the mailbox to see if anything needs to be expunged.
-* autoexpunge_max_mails=<number>: Mails are expunged until mail count is at ``autoexpunge_max_mails`` or below.
-  After these messages are removed, autoexpunge will then try to expunge mails
-  based on the ``autoexpunge`` setting.
+See also: :ref:`setting-namespace-mailbox_autoexpunge_max_mails`.
 
-  .. versionadded:: v2.2.25
 
-  .. code-block:: none
+.. _setting-namespace-mailbox_autoexpunge_max_mails:
 
-    namespace inbox {
-      # the namespace prefix isn't added again to the mailbox names.
-      #prefix = INBOX.
-      inbox = yes
-      # ...
-
-      mailbox Trash {
-	auto = no
-	special_use = \Trash
-      }
-      mailbox Drafts {
-	auto = no
-	special_use = \Drafts
-      }
-      mailbox Sent {
-	auto = subscribe # autocreate and autosubscribe the Sent mailbox
-	special_use = \Sent
-      }
-      mailbox "Sent Messages" {
-	auto = no
-	special_use = \Sent
-      }
-      mailbox Spam {
-	auto = create # autocreate Spam, but don't autosubscribe
-	special_use = \Junk
-      }
-      mailbox virtual/All { # if you have a virtual "All messages" mailbox
-	auto = no
-	special_use = \All
-      }
+``namespace/mailbox/autoexpunge_max_mails``
+-------------------------------------------
+
+.. versionadded:: v2.2.25
+
+- Default: ``0``
+- Values:  :ref:`uint`:
+
+Mails are autoexpunged until mail count is at or below this number of
+messages.
+
+Once this threshold has been reached,
+:ref:`setting-namespace-mailbox_autoexpunge` processing is done.
+
+
+.. _setting-namespace-mailbox_special_use:
+
+``namespace/mailbox/special_use``
+---------------------------------
+
+- Default: <empty>
+- Values:  :ref:`string`:
+
+Space-separated list of SPECIAL-USE
+(`RFC 6154 <http://www.faqs.org/rfcs/rfc6154.html>`_) flags to broadcast for
+the mailbox.
+
+There are no validity checks, so you could specify anything you want here, but
+it's not a good idea to use other than the standard ones specified in the RFC.
+
+.. Note::
+
+  Due to a bug in Dovecot v2.2.30+ if special-use flags are used, SPECIAL-USE
+  needs to be added to post-login CAPABILITY response as RFC 6154 mandates. You
+  can do this with ``imap_capability = +SPECIAL-USE``
```

Comment:
This seems to have been a bug in v2.2.30 .. v2.2.33:

        - v2.2.30+: IMAP stopped advertising SPECIAL-USE capability

Maybe we just remove this paragraph entirely? Few people are likely using those versions.

Reply:
Let's leave it in for the moment, with the correction/addition.  Right now, I have the rough rule that documentation should handle all changes since 2.2.0 - we can revisit this for the next major version release.